tests/system/error.approved.json

```diff
@@ -31,6 +31,9 @@
             "us": 1494342245999000
         },
         "@timestamp": "2017-05-09T15:04:05.999Z",
+        "event": {
+            "ingested": "2017-05-09T15:04:05.998Z"
```

Comment:
The timestamps look odd - have you set them once manually? (They should be some date yesterday)

Reply:
Good eye - yes, I set them manually